src/codeAnalysis/GlobalSuppressions.cs

```diff
@@ -52,3 +52,6 @@
 
 // FxCop warning suppression for uninstantiated TestFixture classes
 [assembly: SuppressMessage("Microsoft.Performance", "CA1812: Avoid uninstantiated internal classes", Scope = "module", Justification = "CA1812 will be thrown for every file in the test project. This is mentioned here: dotnet/roslyn-analyzers#1830")]
+
+// Code quality
+[assembly: SuppressMessage("CodeQuality", "IDE0076:Invalid global 'SuppressMessageAttribute'", Justification = "Affect predefined supressions.")]
```

Comment:
What is the reason for adding this suppression?

Reply:
The suppression file itself show many information messages after enabling analyzer and open the file.

Reply:
I think this can be removed after doing #16791.